relay/paseo/src/lib.rs

```diff
@@ -273,7 +277,11 @@ impl pallet_preimage::Config for Runtime {
 }
 
 parameter_types! {
-	pub EpochDuration: u64 = EPOCH_DURATION_IN_SLOTS as u64;
+	pub EpochDuration: u64 = prod_or_fast!(
+		EPOCH_DURATION_IN_SLOTS as u64,
+		2 * MINUTES as u64,
+		"PAS_EPOCH_DURATION"
+	);
```

Comment:
Not sure if these values are the correct ones

Reply:
No, we need to revert these changes to what we had previously